### PR TITLE
update test

### PR DESCRIPTION
**Pull request by stephenscruggs164**

### CODE REVIEW

File: src/test/java/com/company/MethodsTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    // create an object
+    Methods test = new Methods();
+
+    @Test //-- this is checking for a zero reminder
+    public void modulusSuccess() {
+
+// assertEquals(0,test.modulus(3,2));
+        assertEquals(0, test.modulus(6, 3));
+    }
+
```
Comment (mbsnapit) [inline src/test/java/com/company/MethodsTest.java]: Add a test that passes in numbers that will have a remainder

File: src/test/java/com/company/MethodsTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        int[] num3 = {};
+        int[] num4 = {1, 2, 3, 4};
+        int[] num5 = {2, 3, 4};
+        int[] num6 = {3, 4};
+        int[] num7 = {2, 4, 6, 5, 3, 1, 8, 9};
+        int[] num8 = {10};
+
+//     assertEquals(4, test.missingNumber(num4));
```
Comment (CarloSantos11) [inline src/test/java/com/company/MethodsTest.java]: Remove commented code if it does not describe the code 👍

File: src/test/java/com/company/MethodsTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+// HashSet<Integer> nums = new HashSet<Integer>();
+
+        int[] num1 = {1, 1, 2, 3, 4, 5};
+        int[] num2 = {1, 2, 3, 4, 5};
+
+        assertEquals(1, test.findDuplicatesUsingHashSet(num1));
+// assertNotEquals(1,test.findDuplicatesUsingHashSet(num2));
+    }
+
```
Comment (mbsnapit) [inline src/test/java/com/company/MethodsTest.java]: remove commented code, and int[] num2
add a test that doesnt have a duplicate

File: src/test/java/com/company/MethodsTest.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+     assertEquals(0, test.missingNumber(num4));
+//     assertEquals(0, test.missingNumber(num5));
+//     assertEquals(0, test.missingNumber(num6));
+//     assertEquals(7, test.missingNumber(num7));
+//     assertEquals(1, test.missingNumber(num8));
+
+    //assertEquals(Expecting ARRAY LENGTH, test.missingNumber(num4));
+
+    }
+
```
Comment (mbsnapit) [inline src/test/java/com/company/MethodsTest.java]: write a separate unit test for each of the arrays you created in here
remove commented code

File: src/test/java/com/company/MethodsTest.java [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MethodsTest {
+
+    // create an object
+    Methods test = new Methods();
+
+    @Test //-- this is checking for a zero reminder
```
Comment (CarloSantos11) [inline src/test/java/com/company/MethodsTest.java]: Create tests for all cases. Positive case. Negative case. Edge case. 
A positive case would be when the method does what it is supposed to do. Return 0 when evenly divided and return -1 when there is a remainder.
A negative case would be something like dividing by a number greater than the actual number you are dividing.
An edge case would be something like dividing by 0.
